Cargo.toml

```diff
@@ -86,6 +87,62 @@ which = "6.0.3"
 windows = "0.58.0"
 z3 = "0.12.1"
 
+
+[workspace.lints.rust]
+# Forbid
+unexpected_cfgs = "forbid"
+
+# Deny
+
+# Warn
+
+# Allow
+incomplete_features = "allow"
+ambiguous_glob_reexports = "allow"
+
+
+[workspace.lints.clippy]
+# Forbid
+
+# Deny
+all = { level = "deny", priority = -1 }
+pedantic = { level = "deny", priority = -1 }
+cargo_common_metadata = "deny"
+
+# Warn
+cargo = { level = "warn", priority = -1 }
+
+# Allow
+unreadable_literal = "allow"
+type_repetition_in_bounds = "allow"
+missing_errors_doc = "allow"
+cast_possible_truncation = "allow"
+used_underscore_binding = "allow"
+ptr_as_ptr = "allow"
+missing_panics_doc = "allow"
+missing_docs_in_private_items = "allow"
+module_name_repetitions = "allow"
+ptr_cast_constness = "allow"
+negative_feature_names = "allow"
+unsafe_derive_deserialize = "allow"
+similar_names = "allow"
+too_many_lines = "allow"
+into_iter_without_iter = "allow"        # Broken
+borrow_as_ptr = "allow"                 # Till they fix this buggy lint in clippy
```

Comment:
Can we check if this still applys?

Reply:
was gonna check, for now i just wanted to copy paste and unify before tweaking stuff around